--- trunk/mingw-w64-headers/crt/stdio.h	2012/07/04 13:06:41	5161
+++ trunk/mingw-w64-headers/crt/stdio.h	2012/07/05 11:57:34	5166
@@ -323,6 +323,11 @@
   return __mingw_vsprintf( __stream, __format, __local_argv );
 }
 
+#if 0
+/* Disabled due issue of definition in POSIX way in libmingwex,
+   which could cause troubles on scenarios venure expects those
+   function in MS formatter style.
+   So simplest way for now, disable it.  */
 __mingw_ovr
 __attribute__ ((__format__ (gnu_printf, 2, 3))) __attribute__((nonnull (1,2)))
 int asprintf(char **__ret, const char *__format, ...)
@@ -340,6 +345,7 @@
 {
   return __mingw_vasprintf( __ret, __format, __local_argv );
 }
+#endif
 /* #ifndef __NO_ISOCEXT */  /* externs in libmingwex.a */
 __mingw_ovr
 __attribute__((__format__ (gnu_printf, 3, 4))) __MINGW_ATTRIB_NONNULL(3)
